pkg/address/address.ts

```diff
@@ -1,9 +1,12 @@
 import * as bitcoin from "bitcoinjs-lib"
+import { PrivateKey, key } from "bitsharesjs"
+import { ChainConfig } from "bitsharesjs-ws"
+import crypto from "crypto-random-string"
 
 /**
  * An interface for a privatekey/address pair
  */
-export interface BTCAddress {
+export interface Address {
```

Comment:
Should we really rename this to a generic name?
Do we actually use Address for bitshares?

Reply:
Well, currently not. The address generator is for testing purposes only. However, it is possible to add new key pairs to a given Bitshares account and you should do that regularly.